### DISCUSSION TITLE
Slate 和 ProseMirror 简单对比

### DISCUSSION THREAD

**Discussion by Xheldon** [Xheldon/x_blog#109] [Announcements]
https://www.xheldon.com/tech/prosemirror-comparison-with-slatejs.html 

 Slate 和 Prosemirror 各用过两年后的简单对比总结

**Comment by T-miracle** [comment]
首先感谢博主的翻译！

在PM、Slate、lexical和lexical中考虑了很久，最后还是挑选了PM，主要还是看重他配套的CM了，就是文档有点太晦涩了，看了example-setup示例稍微理解了点，但总体还挺懵逼的，希望能在一个月内弄懂点【捂脸笑哭】

**Comment by Xheldon** [comment]
> @T-miracle
> 首先感谢博主的翻译！
> 
> 在PM、Slate、lexical和lexical中考虑了很久，最后还是挑选了PM，主要还是看重他配套的CM了，就是文档有点太晦涩了，看了example-setup示例稍微理解了点，但总体还挺懵逼的，希望能在一个月内弄懂点【捂脸笑哭】

奥利给~ ;-)

**Comment by totorofly** [comment]
相比之下还是更愿意选择一个稳定API文档完善的项目，我也选了PM。

**Comment by djkloop** [comment]
lexical 这个编辑器如何？不知道作者有没有用过这个....好像挺新的

**Reply by Xheldon** [reply]
听说过，Facebook 出的，各个端都有适配器，主打是多端通用应该是
